modules/javafx.web/src/main/native/Source/WTF/CMakeLists.txt

```diff
@@ -3,7 +3,6 @@ include(WebKitCommon)
 set_property(DIRECTORY . PROPERTY FOLDER "WTF")
 
 add_subdirectory(wtf)
-
```

Comment:
It looks like when you reverted some debugging changes, you removed an extra blank line. Since this file is otherwise now unchanged from master, can you revert the whole file (e.g., by using `git checkout master -- FILENAME`)?

Reply:
Reverted the file to be same as master.